torch/csrc/distributed/autograd/rpc_messages/autograd_backward_failure_req.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/message.h>
+#include <torch/csrc/distributed/rpc/rpc_command_base.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+// Used to notify other workers when there is an autograd error.
+class TORCH_API AutogradBackwardFailureReq : public rpc::RpcCommandBase {
```

Comment:
nit: call this `DistautogradFailureReq` since AutogradBackward might be confused with the local autograd engine.